test/unittests/statetest_loader_tx_test.cpp

```diff
@@ -46,7 +46,7 @@ TEST(statetest_loader, tx_eip1559)
 {
     constexpr std::string_view input = R"({
         "input": "b0b1",
-        "gas": "9091",
+        "gas": "0x9091",
```

Comment:
Why was this passing before?

Reply:
Because we assumed earlier that every field is hex string. Now it can be both and hex has to be prefixed with `0x`

Reply:
This should be also changed probably for bytes, but I don't would prefer to make a separated PR for this

Reply:
Yes I see base was fixed at 16, and now it is auto-detect. Can we add more unit tests for that (to have multiple version of the inputs)?

Reply:
ok

Reply:
Nice! Can you add also octal inputs because `stoull` supports that too.

Reply:
Added one test each.